Azure Service Bus bulk support documentation (#3054)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md

> Note that all times are populated by the server and are not adjusted for clock skews.

## Sending and receiving multiple messages

> Azure Service Bus natively supports sending and receiving multiple messages in a single operation. 

Reviewer: ```suggestion
Azure Service Bus supports sending and receiving multiple messages in a single operation using the bulk Pub/sub API. 
```
Reason: We should remove the term "natively" since it needs to be explained and does not add anything. Just say "Azure Service Bus supports sending and receiving multiple messages in a single operation." No native to explain, as we do not do this with other components. (e.g. We do not say "SQL Server natively supports transactions)

### Configuring bulk publish

To set the metadata for bulk publish operation, set the query parameters on the HTTP request or the gRPC metadata as documented [here]({{< ref pubsub_api >}})

| Metadata | Default |
|----------|---------|

> | `metadata.maxBulkPubBytes` | `131072` (128 KiB) |

Reviewer: These fields needs to be added to the component spec example at the start as well as into the table of values. Putting them here does not make them discoverable. The table is the source of truth for the spec. 
Same comment as the Kafka docs pR https://github.com/dapr/docs/pull/3071

Reviewer: I see the same issue on needing the table in the Kafka Docs PR

Author: > So this is not a part of the component spec. A link will be added here with https://github.com/dapr/docs/pull/3077 that will show how to use this configuration, it will be a part of the response from /dapr/subscribe.

Same as other.

### Configuring bulk subscribe

> When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)

Reviewer: There is a TODO in this sentence

Author: Yes, TODO will be updated once #3077 is merged.

> When subscribing to a topic, you can configure `bulkSubscribe` options. Please refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)

Reviewer 2: ```suggestion
When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)
```

> When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)

Reviewer 2: ```suggestion
When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. Learn more about [the bulk subscribe API]({{< ref pubsub-bulk.md >}}).
```

| Configuration | Default |
|---------------|---------|

> | `maxMessagesCount` | `100` |

Reviewer: These field needs to be added to the component spec example at the start as well as into the table of values. Putting them here does not make them discoverable. The table is the source of truth for the spec.

Author: So this is not a part of the component spec. A link will be added here with #3077 that will show how to use this configuration, it will be a part of the response from `/dapr/subscribe`.
